### DISCUSSION TITLE
Why is there not the option to loose timings?

### DISCUSSION THREAD

**Discussion by ahahfilip** [hanai3Bi/Switch-OC-Suite#42] [Q&A] [answered]
Hi! First thanks for your work :)

I noticed that in "online loader configurator" the default timings for "primary timing" are 18 - 18 - 42, and the others are lower (tighter).
One doubt is: at what frequency this primary timing is default? Usually the default change with its frequency.
A second doubt is: assuming they're primary timing default at 1.6 GHz, why is there not the possibility to loose timings? Take this case, if I increase my frequency, to 1.9962 MHz for example, I want to increase timings as well (say 19 - 19 - 45 ) and just after a stability test I would slowly tighten everything (for example trying to go back to 18-18-42 or even lower).

Can you answer these questions please?

PS: Is there not an Option for the CAS (tCAS/tCL/CL) timing?

**Comment by hanai3Bi** [comment]
1. Timing will stay same unless you manually adjust with configurator. Loosening timings that are adjustable in configurator has no affect in helping with higher clocks, so no need to loosen.

2. In public version of OCS, you cannot adjust CL. (yes you need to loosen CL for higher frequencies). There is reason for this and if you are curious why, dm me at discord

**Reply by hanai3Bi** [reply, marked as answer]
this is not desktop ram oc, so it's slightly different.
